extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -262,28 +252,20 @@ public function getContributorType( $row ) {
 		return self::CONTRIBUTOR_TYPE_USER;
 	}
 
-	public function getVotes() {
-		if ( !empty( $this->votes ) ) {
-			return $this->votes;
-		}
-
+	public function saveVotes() {
 		$pageIds = array_keys( $this->getPages() );
 
-		$dumper = $this;
 		$dbh = wfGetDB( DB_SLAVE );
 		( new \WikiaSQL() )
 			->SELECT_ALL()
 			->FROM( self::TABLE_VOTE )
 			->WHERE( 'article_id' )->IN( $pageIds )
 			->runLoop( $dbh, function ( &$pages, $row ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$pages". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)